Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM nginx:1.14
+FROM nginx:1.14-alpine
 
 COPY ./dist/ /usr/share/nginx/html/
 COPY ./nginx.conf /etc/nginx/nginx.conf
 COPY ./run.sh /usr/share/run.sh
 
 ENV TZ=Asia/Shanghai
 
-RUN chmod +x /usr/share/run.sh 
+RUN set -ex \
+    && apk add --no-cache bash \
+    && chmod +x /usr/share/run.sh
```

Comment:
Could you help to adding `    && cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime \
    && echo "Asia/Shanghai" > /etc/timezone \` here?